content/webapp/utils/iiif/v3/index.ts

```diff
@@ -450,22 +452,25 @@ export function checkIsTotallyRestricted(
 }
 
 export function transformCanvas(canvas: Canvas): TransformedCanvas {
-  const imageService = getImageService(canvas);
+  const imageService = getImageService(canvas); // TODO possibly redo how this is done
```

Comment:
More info here might be useful for posterity? (or is it one of the tickets we could link to?)

Reply:
This is part of one of the issues, I'll remove this comment